lib/govuk_publishing_components/presenters/select_with_search_helper.rb

```diff
@@ -22,6 +22,7 @@ def initialize(local_assigns)
         @select_helper = SelectHelper.new(local_assigns.except(:options, :grouped_options))
         @options = local_assigns[:options]
         @grouped_options = local_assigns[:grouped_options]
+        @include_blank = local_assigns[:include_blank]
```

Comment:
## Question: is `include_blank` only applicable to grouped options?

`include_blank` only applies to grouped options right now.

When a blank/placeholder is needed on 'simple' options, it's manually added in the view. For example here, where we prepend a `[nil]` option:

https://github.com/alphagov/whitehall/blob/379ba86e8aaa381f17a0e5578a3231306444d3fc/app/views/admin/corporate_information_pages/_standard_fields.html.erb#L12

Do we want to take the opportunity to make this behaviour consistent? i.e. remove instances of manually adding `[nil]` in the view, and have the `SelectWithSearchHelper` add the blank/placeholder option instead?

Reply:
Yea i think you're right i think it's a good opportunity to do it. The only thing is that the native govuk-select doesn't have the option to so we will still have to use it for that